composer.json

```diff
@@ -58,6 +68,7 @@
         ],
         "analyse": [
             "phpstan analyse --level=7 src/ tests/",
+            "composer-require-checker",
             "security-checker security:check"
```

Comment:
A failure in phpstan leads to not executing the other script. Because phpstan is the most likely to fail I suggest another order of this scripts